stories/Components/Forms/Radio/radio.scss

```diff
@@ -40,5 +40,9 @@
       transform: translateY(-50%);
       width: 6px;
     }
+
+    &:focus-visible + label .radio_border {
```

Comment:
@Shubham-Sharma-77 we don't need `.radio_border` class here. Directly use `span` to style the radio.
```
&:focus-visible + label span {
 ...
}
```
Also do the change above for `:checked` property and remove the `radio_border` class.

Reply:
@shishirphadke Ok